.babelrc

```diff
@@ -1,6 +1,4 @@
 {
-    "extends": "./node_modules/dash-components-archetype/config/babel/babelrc",
-    "plugins": [
-        "styled-jsx/babel"
-      ]
-}
+    "presets": ["env", "react"],
+    "plugins": ["transform-object-rest-spread", "styled-jsx/babel"]
```

Comment:
In our `dash-component-boilerplate`, we don't have this `plugins` anymore. Do we still need it? If so, should we add it to the `dash-component-boilerplate`?

Reply:
We definitely need `styled-jsx/babel` here because of it's dependency in the `Tabs` component. `transform-object-rest-spread` is also needed if we want to use the [object spread operator](https://babeljs.io/docs/en/babel-plugin-transform-object-rest-spread/)